bike-erp/backend/src/main/routes/component_routes.ts

```diff
@@ -21,7 +21,7 @@ router.get("/", authenticateToken, (req, res) => {
 });
 
 //Backend endpoint to get all the component types in categories
-router.get("/componentTypes", authenticateToken, verifyRole([Role.ADMIN, Role.MANAGER, Role.EMPLOYEE]), (req, res) => {
+router.get("/componentTypes", authenticateToken, verifyRole([Role.ADMIN, Role.MANAGER, Role.EMPLOYEE, Role.CUSTOMER]), (req, res) => {
```

Comment:
If all roles are allowed than you can remove the verify role method

Reply:
@nasaku898 this was done to fix a bug, should we keep it like this so we know what is affected or do I remove the verify?